run_exp.py

```diff
@@ -34,18 +34,20 @@
 makefolder(trace_folder)
 makefolder(log_folder)
 
-t = traceloader(trace_list, trace_folder, 1, 1)
-trace_info=t.retinfo()
+traceloader = TraceLoader(trace_list, trace_folder, 1, 1, 1)
```

Comment:
Do not just pass in number. Please use bool variable and add them within the configuration(like config.toml) and test the combination of different choice to see if it work as you think